stream-chat-android-client/src/main/java/io/getstream/chat/android/client/notifications/handler/NotificationHandler.kt

```diff
@@ -14,6 +22,7 @@ public interface NotificationHandler {
      * @return False if notification should be handled internally.
      */
     public fun onChatEvent(event: NewMessageEvent): Boolean {
+        ChatClient
```

Comment:
Was it added by error?

Reply:
My bad, I was testing something. Thanks!